datalad/interface/common_opts.py

```diff
@@ -148,4 +148,14 @@
     [PY: Multiple dependencies can be given as a list of sibling names
     PY][CMD: This option can be given more than once to configure multiple
     dependencies CMD]""",
+    action='append',
+    constraints=EnsureStr() | EnsureNone())
+
+publish_by_default = Parameter(
+    args=("--publish-by-default",),
+    metavar='REFSPEC',
+    doc="""add a refspec to be published to this sibling by default if nothing
+    specified.""",
+    constraints=EnsureStr() | EnsureNone(),
     action='append')
```

Comment:
I believe that it might need multiple values eg I push all refs and all tags, but sketching that manually is really not user friendly... I thought i had had somewhere an issue to describe common possible publication schemes. But I guess I will try and see first